infra/cifuzz/fuzz_target_test.py

```diff
@@ -87,5 +88,32 @@ def test_with_invalid_error_string(self):
     self.assertIsNone(self.test_target.get_test_case(' Example crash string.'))
 
 
+class DownloadLatestCorpusUnitTest(unittest.TestCase):
```

Comment:
I think there are some better ways this test could be written.
Relying on arduinojson existing is not good.
Instead write a test to ensure that a get request is done on the right URL.
And have another test ensure that when get returns what we expect, the code does what we expect etc.
We need to use mock when possible to avoid tests like this.

Reply:
right, if it uses any external resources, it is an integration test, not a unit test

Reply:
Edited.